atlasdb-jepsen-tests/src/main/java/com/palantir/atlasdb/jepsen/events/InfoEvent.java

```diff
@@ -40,7 +40,7 @@
 
     public abstract long time();
 
-    public abstract Optional<String> value();
+    public abstract Optional<Object> value();
```

Comment:
That is correct, I was getting an exception when trying to cast it to String. I think in general the way that the crashing nemesis is written is not exactly how nemeses should look like, in the sense that it does work, but cannot be composed with the compose nemesis to combine it with others. But that "fix" is definitely a stretch goal as the functionality is what we care about.